### PR TITLE
add test for uuid

### PR DESCRIPTION
**Pull request by yuval-k**

### CODE REVIEW

File: test/common/http/filter/transformer_test.cc [hunk 1]
```diff
@@ -275,5 +276,31 @@ TEST(Transformer, transformBodyNotSet) {
...
 
+TEST(Transformer, transformWithHyphens) {
+  TestHeaderMapImpl headers{
+      {":method", "GET"},
+      {":path", "/accounts/764b0f0f-7319-4b29-bbd0-887a39705a70"}};
+  Buffer::OwnedImpl body("{}");
+
+  envoy::api::v2::filter::http::Extraction extractor;
+  extractor.set_header(":path");
+  extractor.set_regex("/accounts/([\\-._[:alnum:]]+)");
```
Comment (talnordan) [inline test/common/http/filter/transformer_test.cc:287]: Consider using a stricter regex representation of a UUID.

**Review by talnordan — COMMENTED** [review]
LGTM. See minor optional suggestion.

File: test/common/http/filter/transformer_test.cc (hunk 1 above)
Comment (yuval-k) [reply test/common/http/filter/transformer_test.cc:287]: this is actually the same regex we use in gloo; i should probably change the name of the test

Comment (yuval-k) [reply test/common/http/filter/transformer_test.cc:287]: actually the name of the test is good,i'll just add some port characters to it

**Review by talnordan — APPROVED** [review]

**Pull request merged**